front/migrations/20241101_workspace_keys_incident.ts

```diff
@@ -0,0 +1,210 @@
+import { parse } from "csv-parse";
+import * as fs from "fs";
+import { exit } from "process";
+
+import { sendEmailWithTemplate } from "@app/lib/api/email";
+import { getMembers } from "@app/lib/api/workspace";
+import { Authenticator } from "@app/lib/auth";
+import { Workspace } from "@app/lib/models/workspace";
+import { getStripeClient } from "@app/lib/plans/stripe";
+import type { Logger } from "@app/logger/logger";
+import { makeScript } from "@app/scripts/helpers";
+
+const MIN_AMOUNT_FOR_REIMBURSEMENT = 1;
+const CURRENCY = "eur";
+
+const sendIncidentEmailToAdmins = async (
+  workspaceId: number,
+  cost_in_cents: number,
+  reimbursement_in_cents: number,
+  logger: Logger
+) => {
+  const message = {
+    from: {
+      name: "Dust team",
+      email: "team@dust.tt",
+    },
+    subject: `[Dust] Incident 2024/10/31`,
+    body: `<p>On October the 31st, due to a misconfiguration on our end, from 12:44 to 13:20 our API keys to OpenAI, Anthropic, Gemini and Mistral became unavailable to our systems triggering a fallback to your own API keys for assistants interactions within your workspace.</p>
```

Comment:
Let's add an "Hello,"

Reply:
The hello is done by the template (I tested)

Reply:
gotcha :+1: